### ISSUE TITLE
Installing error - Architacture mismatch?  

### ISSUE THREAD

**Issue by soyludev** [trufflesuite/truffle#4981]
## Issue

Installation is not getting completed due to numerous amount of error. That list of errors confused me. Please check the log that shared below.  

## Steps to Reproduce
npm install truffle -g

## Actual Results

Tail of the log : https://pastebin.com/WkmicLjn

## Environment

* Operating System: MacOS X 10.15.7
* Ethereum client: none
* Truffle version (`truffle version`): none 
* node version (`node --version`): v16.14.2
* npm version (`npm --version`): 8.5.0

**Comment by eggplantzzz** [comment]
Hey @soyludev, it seems people using version 8 of npm are experiencing problems installing Truffle when their build tools are a bit out of date: in your case, xCode on mac. So make sure you update xCode to the most current version. You might need to update your OS to do this as you are using 10.x. I had to update to 11.x before I could get the latest.

You could also downgrade to npm version 6 as well as a quick and dirty workaround. Let me know how it goes!

**Comment by soyludev** [comment]
@eggplantzzz thanks for your answer. Honestly, xCode is not installed in my system at the moment. Let me try again with xCode installed system.

**Comment by soyludev** [comment]
Unfortunately, did not work out

**Comment by soyludev** [comment]
downgrading npm did work for me! Thanks @eggplantzzz for your help. 

> You will need to make sure you are running the most up to date version of Xcode. I'm using 13.2.1. You may also need to update your OS to 11.x, this is what I had to do on my local machine to upgrade Xcode which allowed it to install using npm version 8. You can also, like I said in the previous response, downgrade to npm version 6 (`npm install -g npm@6`) and try again.
